pkgs/by-name/st/stm32cubemx/package.nix

```diff
@@ -52,6 +52,11 @@ let
 
       cat << EOF > $out/bin/${pname}
       #!${stdenvNoCC.shell}
+      updater_xml="\$HOME/.stm32cubemx/thirdparties/db/updaterThirdParties.xml"
+      if [ -e "\$updater_xml" ] && [ ! -w "\$updater_xml" ]; then
+        echo "Warning: Unwritable \$updater_xml prevents CubeMX software packages from working correctly. Fixing that."
+        chmod u+w "\$updater_xml"
```

Comment:
```suggestion
        (set -x; chmod u+w "\$updater_xml")
```

Reply:
I've applied your suggestion and rebased the branch on top of `master` again.